tools/clang/include/clang/Sema/Sema.h

```diff
@@ -1514,7 +1514,9 @@ class Sema {
     NameClassification(ParsedType Type) : Kind(NC_Type), Type(Type) {}
 
     NameClassification(const IdentifierInfo *Keyword)
-      : Kind(NC_Keyword), Keyword(Keyword) { }
+      : Kind(NC_Keyword), Keyword(Keyword) {
+      (void)Keyword;
```

Comment:
Instead of this, I commented out the member and the initialization here. It's just those two lines that remain active.